### PR TITLE
Replace Throw `CacheException::class` for message `LoggerInterface::class`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,17 @@ To work with values in a more efficient manner, batch operations should be used:
...
+## Implemented `Psr\Log\LoggerInterface::class`
+
+You can use our package [yiisoft\Log](https://github.com/yiisoft/log)
+
+```php
+$cache = new \Yiisoft\Cache\Db\DbCache($db, $table, $gcProbability);
+$cache->setLogger(new \Yiisoft\Log\Logger());
+```
+
+This allows you to log cache operations, when ocurring errors, etc.
```
Comment (samdark) [inline README.md:105]:
```suggestion
## Additional logging

In order to log details about failures you may set a logger instance. It should be `Psr\Log\LoggerInterface::class`. For example, you can use [yiisoft\Log](https://github.com/yiisoft/log):

```php
$cache = new \Yiisoft\Cache\Db\DbCache($db, $table, $gcProbability);
$cache->setLogger(new \Yiisoft\Log\Logger());
```
```

File: src/DbCache.php [hunk 2]
```diff
@@ -35,6 +37,8 @@
  */
 final class DbCache implements CacheInterface
 {
+    use LoggerAwareTrait;
```
Comment (samdark) [inline src/DbCache.php:40]: PSR logger should be added to composer.json suggest section.

File: README.md (hunk 1 above)
Comment (terabytesoftw) [reply README.md]: done.

File: src/DbCache.php (hunk 2 above)
Comment (terabytesoftw) [reply src/DbCache.php:40]: done.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**